docs/user/configuration.rst

```diff
@@ -1,94 +0,0 @@
-Configuring QCoDeS
-==================
-
-
-QCoDeS uses a JSON based configuration system.
-The configuration is modeled after the module structure plus some higher level switches.
-
-Config files
-------------
-QCoDeS config files are read in the following order:
-
- - Package defaults
- - User's home directory
- - QCODES_CONFIG environment variable
- - Current working directory
-
-This means that something specified in your current working directory will overwrite the package defaults.
-
-Config files are backed by a json-schema, meaning that they will be validated before use, and if an error occurs the default settings are kept, and the user is informed.
-
-.. note:: A change in the configuration requires reimporting the package, or restarting the notebook kernel.
-
-
-Default config
---------------
-The following  table explains the default configuration of qcodes.
-
-.. jsonschema:: ../../qcodes/configuration/qcodesrc_schema.json
```

Comment:
the great part of this document on qcodes config is that the full real json schema of the config is rendered as a nice table https://qcodes.github.io/Qcodes/user/configuration.html#default-config - perhaps we can keep that somehow? embed in the notebook or keep a small rst document for that and reference it in the https://qcodes.github.io/Qcodes/examples/Configuring_QCoDeS.html notebook ?

Reply:
I ... don't find this table nice or readable, but it is definitely complete. I am probably worse off looking at it on a much smaller screen than you, but this should be kept in mind for users in general.

Consistent with my earlier replies, it should be merged into the examples/configuring qcodes notebook.

Reply:
> it should be merged into the examples/configuring qcodes notebook

then let's do that in this PR?

Reply:
Yes, I am using this brach/PR for removing the user guide and consolidating its sections into various example notebooks. I don't want to crowd the system too much :D

Reply:
I think removing this means that we can drop the dependency on sphinx-jsonschema

Reply:
Moved this to the api docs and linked to that from the config notebook